packages/shared/src/components/post/PostNavigation.tsx

```diff
@@ -29,6 +30,7 @@ export interface PostNavigationProps extends PostActions {
   onNextPost?: () => unknown;
   className?: PostNavigationClassName;
   children?: ReactNode;
+  post: Post;
```

Comment:
Can you double check these changes in this file?
think it's already in the props, as far as i can tell its not needed

Reply:
Apologies, I was getting the two files mixed up. Cleaning it up now